ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -254,8 +254,8 @@ private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
                 textBlock.Padding = new Thickness(
                     GetPadding(YogaEdge.Left),
                     GetPadding(YogaEdge.Top),
-                    0,
-                    0);
+                    GetPadding(YogaEdge.Right),
```

Comment:
Interesting - is this an unrelated bug?

Reply:
This was the reason the text padding was not working correctly in RTL. I used fuzzy logic (i.e. without clear understanding on how it's supposed to work) and made the thing consistent by not ignoring some values. The padding error was suddenly fixed...

---
In reply to: [223475202](https://github.com/Microsoft/react-native-windows/pull/1994#discussion_r223475202) [](ancestors = 223475202)